using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -1,16 +1,16 @@
 # Adobe Campaign Standard
 
 {% hint style="info" %}
-**Before** you install or configure the Campaign Standard extension, read [_Getting Started_](../../getting-started/create-a-mobile-property.md) and [Configuring a mobile application using Adobe Experience Platform SDKs](https://helpx.adobe.com/campaign/kb/configuring-app-sdk.html).
+**Before** you install or configure the Campaign Standard extension, please read the [getting started guide](../../getting-started/create-a-mobile-property.md) and the [configuring a mobile application using Adobe Experience Platform SDKs guide](https://experienceleague.adobe.com/docs/campaign-standard/using/administrating/configuring-channels/configuring-a-mobile-application.html).
 {% endhint %}
 
 {% hint style="danger" %}
-If you participated in the Campaign Standard beta, to use the new Campaign Standard extension, go to [launch.adobe.com](https://launch.adobe.com), instead of the Experience Platform Launch integration environment, .
+If you participated in the Campaign Standard beta, to use the new Campaign Standard extension, go to the [Adobe Experience Platform Data Collection UI](https://experience.adobe.com/data-collection/), instead of the Experience Platform Data Collection integration environment.
```

Comment:
"(Adobe) Experience Platform Data Collection" is to be avoided, always. While it _is_ in the UI (facepalm), we do not mention it since it sets up "Data Collection" as a fully branded Platform service (it is not).

When referring to the UI, it's just "the Data Collection UI", no "Platform" prefix.